packages/components/src/DataList/components/DataListAction/DataListAction.tsx

```diff
@@ -23,7 +23,7 @@ export function DataListAction<T extends DataListObject>({
     return null;
   }
 
-  const color = destructive ? "critical" : "heading";
+  const color = destructive ? "destructive" : "heading";
```

Comment:
![image](https://github.com/user-attachments/assets/50243543-29ab-4e0f-a637-5f697ba36d85)

Reply:
Local URL for [DataList](http://localhost:6005/?path=/story/components-lists-and-tables-datalist-web--basic)